modules/installation-user-infra-machines-iso.adoc

```diff
@@ -82,6 +83,16 @@ For example, setting
 * If your system has multiple network interfaces, you can add multiple IP address configurations by specifying multiple `ip=` entries.
 * If your system has multiple network interfaces, you can use a combination of DHCP and static IP configurations. You must specify the interface to use, for example `ip=enp1s0:dhcp ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none`.
 * You can provide multiple DNS servers by adding a `nameserver=` entry for each server, such as `nameserver=1.1.1.1 nameserver=8.8.8.8`.
+<6> Optionally, you can bond multiple network interfaces to a single interface
+using the `bond=` option, as described here:
+* The syntax is: `bond=name[:slaves][:options]`.
```

Comment:
Can you use a different word for "slaves" here and in the sentence below? We are getting ready to remove this kind of language from our docs and would like not to introduce another instance now.

Reply:
Okay. I'll change "slaves" into "network_interfaces".

Reply:
Done.